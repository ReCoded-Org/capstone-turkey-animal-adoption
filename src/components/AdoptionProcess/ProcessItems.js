import React from 'react'
import { Col } from 'react-bootstrap';
import "./AdoptionProcess.css"

const ProcessItems = ({ img, text, firstDescription, secondDescription }) => {
   
    return (
        <Col lg={4} className="mb-5">
            <img src={ img } alt="img" className="mb-3"/>
            <p>{ text }</p>
            <span>{ firstDescription }</span> <br />
            <span>{ secondDescription }</span>
        </Col>
    )
}

export default ProcessItems

