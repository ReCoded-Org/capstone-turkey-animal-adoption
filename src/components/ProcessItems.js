import React from 'react'
import { Col } from 'react-bootstrap';
import "./AdoptionProcess.css"

const ProcessItems = ({ process }) => {
   
    return (
        <Col lg={4} className="mb-5">
            <img src={ process.img } alt="img" className="mb-3"/>
            <p>{ process.text }</p>
            <span>{ process.firstDescription }</span> <br />
            <span>{ process.secondDescription }</span>
        </Col>
    )
}

export default ProcessItems
