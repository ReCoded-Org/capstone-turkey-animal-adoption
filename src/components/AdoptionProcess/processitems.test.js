import React from "react";
import renderer from "react-test-renderer";
import ProcessItems from "./ProcessItems.js";

it("renders correctly", () => {
  const tree = renderer.create(<ProcessItems />).toJSON();
  expect(tree).toMatchSnapshot();
});
