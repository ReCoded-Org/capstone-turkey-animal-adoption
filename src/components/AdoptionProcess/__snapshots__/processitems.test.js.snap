// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mb-5 col-lg-4"
>
  <img
    alt="img"
    className="mb-3"
  />
  <p />
  <span />
   
  <br />
  <span />
</div>
`;
